common/persistence/visibility/store/elasticsearch/client/backoff.go

```diff
@@ -0,0 +1,40 @@
+package client
```

Comment:
What is this file?

Reply:
Ahahahah! how did git not show me this in my history